kafka-ui-react-app/src/components/common/Select/LiveIcon.styled.tsx

```diff
@@ -0,0 +1,31 @@
+import styled from 'styled-components';
+import React from 'react';
+
+interface Props {
+  className?: string;
+  selectSize: 'M' | 'L';
+}
+
+const LiveIcon: React.FC<Props> = ({ className }) => {
+  return (
+    <i className={className}>
+      <svg
+        width="16"
+        height="16"
+        viewBox="0 0 16 16"
+        fill="none"
+        xmlns="http://www.w3.org/2000/svg"
+      >
+        <circle cx="8" cy="8" r="7" fill="#FAD1D1" />
+        <circle cx="8" cy="8" r="4" fill="#E61A1A" />
+      </svg>
+    </i>
+  );
+};
+
+export default styled(LiveIcon)`
+  position: absolute;
+  left: 12px;
+  top: 50%;
+  transform: translateY(-36%);
```

Comment:
Why `-36%`?
I think that it should be -50%. (https://www.figma.com/file/ZkvysdRUmt9p2PDNU3a9WU/Kafka-UI?node-id=64%3A3950)

Reply:
I'm not sure how it works, but 50% doesn't make centered:
![image](https://user-images.githubusercontent.com/31561808/134154205-b277c8c8-b1df-40ad-88b2-0740949a4ffa.png)

And 36 does:
![image](https://user-images.githubusercontent.com/31561808/134154149-3116985e-a438-4315-8c37-da6e76daa0a6.png)

Reply:
try to add `line-height: 1;`.